.pre-commit-config.yaml

```diff
@@ -53,8 +53,8 @@ repos:
     rev: 37.150.1
     hooks:
       - id: renovate-config-validator
-  - repo: https://github.com/Yelp/detect-secrets
-    rev: v1.4.0
-    hooks:
-    -   id: detect-secrets
-        args: ['--baseline', '.secrets.baseline']
+#  - repo: https://github.com/Yelp/detect-secrets
```

Comment:
Didn't have the energy to figure out whats broken with my latest package updates with this - will tackle in a separate PR